backend/stories/crud.py

```diff
@@ -55,3 +58,21 @@ def create_story_symptoms(
     db.add_all(db_symptoms)
     db.commit()
     return db_symptoms
+
+
+def create_travels(db: Session, travels: List[schemas.TravelCreate]):
+    db_travels = [models.Travel(**travel.dict()) for travel in travels]
+    db.add_all(db_travels)
+    db.commit()
+    return db_travels
+
+
+def update_travel(db: Session, travel: schemas.Travel):
```

Comment:
Not _precisely_ - the story gets a `CreateStory` pydantic model, which doesn't have an `id`. Back in the day I couldn't find a way to make it recieve a plain `Story` and not have pydantic explode. I could try again, but I'm not sure if it's worth it.

Reply:
You are right, I hadn't noticed that the `update_story` receives the `id` in a separate parameter

All the same,  you could just simply keep the signature of `update_story`, and abstract the method and parameterize it with the model **and** the id, where in travel you would pass `travel.id` and in story the `story_id` given by parameter, right? or I'm loosing something?

Look at this and tell me what you think
```python

def update_story(db: Session, story_id: int, story: schemas.StoryCreate):
    return update(story_id, story, models.Story, db)

def update_travel(db: Session, travel: schemas.Travel):
    return update(travel.id, travel, models.Travel, db)

def update(model_id: int, item: BaseModel, model: Base, db: Session):
    item_as_dict = dict(item)
    db_item = (
        db.query(model).filter(model.id == model_id).first()
    )
    for k, v in item_as_dict.items():
        setattr(db_item, k, v)
    db.commit()
    return db_item
```
The travel test passes and running it it updates the story aswell

Reply:
Oh yeah! Sorry, my dizzy brain was trying to put it together the other way around (simplifying to the `travel_update` instead of the `story_update`). I'll add it rightaway!